src/genconfig.sh

```diff
@@ -5,8 +5,13 @@ OUTPUT_FILE=$SRC_DIR/kernel-config.h
 FEATURE_TEST_DIR="$SRC_DIR/configure-tests/feature-tests"
 FEATURE_TEST_FILES="$FEATURE_TEST_DIR/*.c"
 SYMBOL_TESTS_FILE="$SRC_DIR/configure-tests/symbol-tests"
+KERNEL_VERSION=`uname -r`
```

Comment:
Use `$()`

Reply:
I was following the convention used in other parts of the script, but I can change that, yeah.